tools/test.mk

```diff
@@ -977,29 +977,29 @@ test-all: clean
 
 
 test-size-all:
-	make test-size SIGN=NONE LIMIT=4776
+	make test-size SIGN=NONE LIMIT=4917
```

Comment:
Are these the actual increased values? Would you like me to recompute updated actuals? On my Ubuntu 24 box I can reproduce the values that match GitHub CI.